src/witan/app/handler.clj

```diff
@@ -161,7 +161,12 @@
                                               version :- java.lang.Long
                                               category :- String]
                                 (forecast/input-data {:id id :version version :category category :user-id (:identity request)}))
-                   (sweet/POST* "/tag" []
+                   (sweet/GET* "/forecasts/:id/:version/input/:category" []
+                                :summary "get input data item"
+                                :path-params [id :- java.util.UUID
+                                              version :- java.lang.Long
+                                              category :- String]
+                                (forecast/input-data {:id id :version version :category category}))                               (sweet/POST* "/tag" []
                                 :summary "Creates a new tag from a forecast id and version"
                                 (not-implemented))
```

Comment:
don't know what happened there, it's a case of an end-of-line disappearing (tag url at the end of the line)